backend/private-graph/graph/schema.resolvers.go

```diff
@@ -5815,6 +5816,17 @@ func (r *queryResolver) Resources(ctx context.Context, sessionSecureID string) (
 		return nil, e.Wrap(err, "error getting resources from redis")
 	}
 
+	resourceSize := size.Of(resources)
+	log.WithContext(ctx).WithFields(
+		log.Fields{
+			"size":            resourceSize,
+			"sessionSecureID": sessionSecureID,
+		}).Info("[Resources] Fetched resources size")
+
+	if resourceSize > MaxDownloadSize {
+		return nil, fmt.Errorf("resource size (%v) exceeds max download size", resourceSize)
```

Comment:
could we check that the frontend shows the existing error popup? 
https://github.com/highlight/highlight/blob/7c3431b68472e118217cd49b5215e2a039b5eaed/frontend/src/pages/Player/Toolbar/DevToolsWindowV2/NetworkPage/NetworkPage.tsx#L501-L521

Reply:
Updated

![Screenshot 2024-08-27 at 10 25 07 AM](https://github.com/user-attachments/assets/3dd0ef4b-7eee-47ce-af74-9d1d2ed13aac)